### ISSUE TITLE
Revert a PR

### ISSUE THREAD

**Issue by felipethome** [bors-ng/bors-ng#578]
Hi! Is there an easy way to revert a PR with bors since the github button `revert` doesn't show in PRs that were merged with `bors r+`?

**Comment by notriddle** [comment]
No, there isn't a command to do that right now. I'd probably want to do [an RFC for adding it, once the process is finalized and everything](https://forum.bors.tech/t/pre-rfc-creating-a-formal-decision-making-process-for-bors-ng/269)...

**Comment by DilumAluthge** [comment]
@felipethome Now that the RFC process is up and running, would you be willing to draft an RFC for this feature request? You can submit a draft RFC here: https://forum.bors.tech/c/development/draft-rfcs

**Comment by kamalmarhubi** [comment]
This may not be relevant anymore? See any recent PR to this repo, eg https://github.com/bors-ng/bors-ng/pull/1147

![image](https://user-images.githubusercontent.com/50936/107584079-d8831580-6bc9-11eb-8eee-64b002c591fc.png)

**Comment by dessskris** [comment]
+1 for this issue.

The PR that @kamalmarhubi posted no longer has the Revert button.

<img width="935" alt="image" src="https://user-images.githubusercontent.com/16229203/162441679-5c12ebfe-0626-448a-b72a-e53831eeb9a8.png">
